extensions/binding/org.eclipse.smarthome.binding.meteoblue/README.md

```diff
@@ -0,0 +1,151 @@
+
+
+## Channels
+
+### Channel Groups
+
+| Group Name       | Description |
+| ---------------- | ----------- |
+| forecastToday    | Today's forecast |
+| forecastTomorrow | Tomorrow's forecast |
+| forecastDay2     | Forecast 2 days out |
+| forecastDay3     | Forecast 3 days out |
+| forecastDay4     | Forecast 4 days out |
+| forecastDay5     | Forecast 5 days out |
+| forecastDay6     | Forecast 6 days out |
+
+### Channels
+
+Each of the following channels is supported in all of the channel groups.
+
+| Channel                  | Item Type          | Description |
+| ------------------------ | ------------------ | ----------- |
+| height                   | Number:Length      | Altitude above sea-level of the location (in meters) |
+| forecastDate             | DateTime           | Forecast date |
+| UVIndex                  | Number             | UltraViolet radiation index at ground level (0-16) |
+| minTemperature           | Number:Temperature | Low temperature |
+| maxTemperature           | Number:Temperature | High temperature |
+| meanTemperature          | Number:Temperature | Mean temperature |
+| feltTemperatureMin       | Number:Temperature | Low "feels like" temperature |
+| feltTemperatureMax       | Number:Temperature | High "feels like" temperature |
+| relativeHumidityMin      | Number             | Low relative humidity |
+| relativeHumidityMax      | Number             | High relative humidity |
+| relativeHumidityMean     | Number             | Mean relative humidity |
+| precipitationProbability | Number             | Percentage probability of precipitation |
+| precipitation            | Number:Length      | Total precipitation (water amount) |
+| convectivePrecipitation  | Number:Length      | Total rainfall (water amount) |
+| rainSpot                 | String             | Precipitation distribution around the location |
+| rainArea                 | Image              | Color-coded image generated from rainSpot |
+| snowFraction             | Number             | Percentage of precipitation falling as snow |
+| snowFall                 | Number:Length      | Total snowfall (calculated) |
+| cardinalWindDirection    | String             | Name of the wind direction (eg. N, S, E, W, etc.) |
+| windDirection            | Number             | Wind direction (in degrees) |
+| minWindSpeed             | Number:Speed       | Low wind speed  |
+| maxWindSpeed             | Number:Speed       | High wind speed |
+| meanWindSpeed            | Number:Speed       | Mean wind speed |
+| minSeaLevelPressure      | Number:Pressure    | Low sea level pressure  |
+| maxSeaLevelPressure      | Number:Pressure    | High sea level pressure |
+| meanSeaLevelPressure     | Number:Pressure    | Mean sea level pressure |
+| condition                | String             | A brief description of the forecast weather condition (e.g. 'Overcast') |
```

Comment:
Maybe worth to mention that the state is a number between 1-17 (and provide the link to https://content.meteoblue.com/nl/service-specifications/standards/symbols-and-pictograms#eztoc14635_1_6 here).

Reply:
Done.